### ISSUE TITLE
React Server Components not supported in NextJS 13

### ISSUE THREAD

**Issue by dotlouis** [iconoir-icons/iconoir#219]
## Prerequisites

* Version:  "iconoir-react": "^5.4.1"
* Are you running from source/main: no
* Are you using a released build: yes
* Operating system: MacOS 12.6.1 M1
* Bits:

* "react": "18.2.0",
* "react-dom": "18.2.0",
* "next": "13.0.5",

## Step to reproduce

Importing a iconoir-react component in any "default" (ie server) component will result in an error

```TSX
import { OpenNewWindow } from 'iconoir-react'

const ServerComponent()  => (
    <OpenNewWindow width="1rem" height="1rem" />
)
```

### Actual behavior

It's possible to bypass it by using a [client component](https://beta.nextjs.org/docs/rendering/server-and-client-components#client-components) instead.
But it means wrapping every icon into a client component, which is not ideal.

```TSX
'use client' // 👈 this does the trick

import { OpenNewWindow } from 'iconoir-react'

const ClientComponent()  => (
    <OpenNewWindow width="1rem" height="1rem" />
)
```

## Any message or error

```
TypeError: react_1.default.createContext is not a function

index.js?46cb:598 Uncaught TypeError: react_1.default.createContext is not a function
    at eval (webpack-internal:///(:3000/sc_server)/./node_modules/iconoir-react/dist/IconoirContext.js:12:42)
    at Object.(sc_server)/./node_modules/iconoir-react/dist/IconoirContext.js ([...]/.next/server/app/page.js:24575:1)
    at __webpack_require__ ([...]/.next/server/webpack-runtime.js:33:43)
    at eval (webpack-internal:///(:3000/sc_server)/./node_modules/iconoir-react/dist/3DAddHole.js:49:24)
    at Object.(sc_server)/./node_modules/iconoir-react/dist/3DAddHole.js ([...]/.next/server/app/page.js:18349:1)
    at __webpack_require__ ([...]/.next/server/webpack-runtime.js:33:43)
    at eval (webpack-internal:///(:3000/sc_server)/./node_modules/iconoir-react/dist/index.js:35:20)
    at Object.(sc_server)/./node_modules/iconoir-react/dist/index.js ([...]/.next/server/app/page.js:31659:1)
    at __webpack_require__ ([...]/.next/server/webpack-runtime.js:33:43)
    at eval (webpack-internal:///(:3000/sc_server)/./src/app/layout.tsx:6:71)
```

## Additional info or screenshots

* [Docs to ServerComponent](https://beta.nextjs.org/docs/rendering/server-and-client-components)

**Comment by sammarks** [comment]
@dotlouis - I'm having some trouble reproducing this one. Can you check out this StackBlitz and let me know if I'm reproducing anything incorrectly?

My understanding is the `index.js` file is a ServerComponent, and both `npm run dev` and `npm run build` work without error.

I also tried this on the latest version of Iconoir and NextJS, and it works fine there too.

Can you see if you can provide a CodeSandbox / StackBlitz sandbox reproducing this issue?

**Comment by sammarks** [comment]
@dotlouis I'm actually having some trouble reproducing this with the versions you mentioned in your issue. Can you check out [this StackBlitz?](https://stackblitz.com/edit/nextjs-acxjcy?file=pages/index.js)

I would also try upgrading to the latest version of Iconoir (I don't think it'll make much of a difference since all we did was change a few icons around, but just in case).

If you're still having issues, can you try to create a reproduction in a CodeSandbox or StackBlitz sandbox, so we can take a deeper dive?

**Comment by paescuj** [comment]
@dotlouis We were able to come up with a reproduction: https://stackblitz.com/edit/nextjs-acxjcy

**Comment by sammarks** [comment]
🎉 Fixed in v6.2.0.

Here's a working reproduction with the latest version: https://stackblitz.com/edit/nextjs-ftmwjd?file=app%2Fpage.js
